res/Sectors/M1105/Extolian.txt

```diff
@@ -41,437 +42,436 @@
+3108 Andre                C777961-C Hi In Hive8 Huma2 Cy O:3209   {  3 } (989-2) [6C1B] - -  - 203 11 Hv  K8 V               
+3109 Aokorurnanglir       C783203-9 Lo                            { -1 } (C11+3) [111A] - K  - 802 11 Hv  K2 V               
+3203 Katana               A858778-A Ag                            {  3 } (46A-2) [5AAC] - K  - 900 11 Na  K2 V               
+3204 (Sauna)              E884000-0 Ba                            { -3 } (A00-1) [0000] - -  - 000  9 --  A7 V               
+3206 Loqaock              E777405-9 Ni Pa                         { -2 } (930-1) [6278] - -  - 300  8 Hv  G5 V               
+3207 Fun                  D879301-E Lo Huma3                      { -1 } (A21+0) [32AF] - -  - 102  7 Hv  K2 V               
+3209 Sispo III            B444726-F Ag Pi Cp                      {  3 } (76B-1) [5A5F] - M  - 414 13 Hv  G2 V               
+0114 Pisis                X7B6698-0 Fl Ni                         { -3 } (651+0) [3381] - -  - 902 12 Cr  M5 V M2 V          
+0115 Iass Rsfii'kr        B463422-A Ni Cp                         {  1 } (933+4) [5568] - K  - 500  4 Cr  G9 V K4 V          
+0116 Skia'mia             D967546-2 Ag Ni Pr                      { -2 } (B43-2) [7342] - -  - 401  9 Cr  G3 V M5 V          
+0117 Carver               C300556-9 Ni Va                         { -1 } (642+4) [4427] - M  - 600  6 Na  M3 V               
+0214 Selmore              B552241-A Lo Po                         {  1 } (611+0) [2317] - -  - 202  8 Na  G3 V               
+0312 Lead                 B405310-9 Lo Va Ic                      {  0 } (A21+0) [137C] - M  - 221 16 Na  K1 V               
+0318 Enfer                X9C4000-0 Fl Ba                         { -3 } (600+1) [0000] - -  - 000  5 --  M4 V               
+0413 Hamius Farms         B642210-7 He Lo Po                      { -1 } (711-1) [1197] - K  - 110  8 Na  F1 V A3 V          
+0420 Pakuucoon            A668402-F Ni Pa Cp                      {  1 } (634+1) [657F] - E  - 100  7 Hv  A5 V M4 V          
+0519 Flandre              C437751-A                               {  1 } (669-3) [687A] - -  - 512 12 Na  K0 V               
+0612 Dahaq                B2006S1-F Na Ni Va                      {  1 } (B56-2) [974F] - K  - 904 10 Hv  K1 III K2 V        
+0613 Lunraot              E878725-9 Ag Pi                         {  0 } (A66+1) [B748] - -  - 104 16 Hv  K1 V               
+0711 Kug                  A581764-E Ri O:0809                     {  3 } (E65-1) [5A4F] - E  - 625 13 Hv  G2 V               
+0715 Olifant              C1108BB-B Na Ph Pi                      {  1 } (B7B+1) [893F] - -  - 104  8 Na  K0 V               
+0719 Ooqeliqfalax         X210698-0 Na Ni Poxi8 Fo                { -3 } (850+1) [3392] - -  R 120  5 Xh  G7 V               
+0813 Ilnuar               E536312-D Lo                            { -1 } (721+3) [826B] - -  - 900  9 Hv  K7 V               
+0814 Ulphlairdeern        C689465-B Ni Re
```

Comment:
The sophont percentages could be tweaked here: (Phlinskwal)6 Ithk4

Reply:
Changed